### DISCUSSION TITLE
Import Math blocks from Markdown

### DISCUSSION THREAD

**Discussion by yuri-becker** [docmost/docmost#526] [Help / Questions] [answered]
Markdown often supports Math blocks, marked by $ e.g. ```$\overline{A\underline{⋁}B}$``` for an in-line one, 
```
$$
\overline{A\underline{⋁}B}
$$
```
for a block one.

Within the blocks, it's the same syntax that Docmost is using.

However, when doing a Markdown import those blocks do not get imported properly and just stay in literal form.

In other words, my imported Markdown ends up as `$$Q=UIsin(Φ)$$` instead of a math block.

To make the transition easier, I would love it if Docmost could import the math blocks correctly. I'm also open to contributing to this project by implementing this myself, if you deem it useful

**Comment by Philipinho** [comment]
I noticed it too. I am working on a fix.

**Reply by Philipinho** [reply]
Just pushed my local copy. (PR https://github.com/docmost/docmost/pull/529).
I will work on finalizing it.

**Reply by Philipinho** [reply, marked as answer]
This is now fixed in v0.6.1.

**Reply by yuri-becker** [reply]
Thank you so much, looking forward to it :)
